corehq/motech/openmrs/finders.py

```diff
@@ -54,6 +55,9 @@ class PatientFinder(DocumentSchema):
     Subclasses must implement the `find_patients()` method.
     """
 
+    # If no patients are found, should a new one be created?
```

Comment:
I assume this is an explanation of the property below, but it does sound like a question to the reader